src/amplify-migration/githubUtils.ts

```diff
@@ -82,3 +83,46 @@ export async function isRepoMigrated(repoName: string): Promise<boolean> {
   const content = Buffer.from(result.data.content, "base64").toString("ascii");
   return content.includes(".amplifyapp.com");
 }
+
+export async function createStagingLiteBranch(repoName: string): Promise<void> {
+  const remoteRepoUrl = `https://github.com/${ORGANIZATION_NAME}/${repoName}.git`;
+  const stgLiteDir = path.join(`${process.cwd()}/../${repoName}-staging-lite`);
```

Comment:
why do we need path join for a single string?

also this seems to be a relative path from the current directory. From which directory are we supposed to execute this script? Is this just a temp folder?

Reply:
hey yup you are right the path is unnecessary! 
the idea was to run it directly in codespaces, so the tree structure will be 
|__ isomer-conversion-script
|__ <repo-name>
|__ <repo-name>-staging-lite

Reply:
d3b2f81870b4fea9ae834b330f407306a927a8d2